src/odemis/model/test/dataflow_test.py

```diff
@@ -159,22 +164,42 @@ def synchronizedOn(self, event):
         if self._sync_event:
             self._sync_event.subscribe(self)
 
-        
+
+class DataFlowRaiseError(model.DataFlow):
+    """Very simple dataflow which raises an error during the start of the data generation."""
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
You can drop this, if there is nothing special to do.

Reply:
If I drop it, I receive the following error when running the test:
```
Error
Traceback (most recent call last):
  File "/home/sabrina/development/odemis/src/odemis/model/test/dataflow_test.py", line 329, in test_df_raise_error
    im = self.df.get()
  File "/home/sabrina/development/odemis/src/odemis/model/_dataflow.py", line 326, in get
    self.subscribe(receive_one_image)
  File "/home/sabrina/development/odemis/src/odemis/model/_dataflow.py", line 335, in subscribe
    with self._lock:
AttributeError: 'DataFlowRaiseError' object has no attribute '_lock'
```
So it looks like it is still necessary in order to have the lock attribute. Same for `_listeners` and` _remote_listeners`. So either I need to leave the super or I need to instantiate these attributes in the init I think. Is it ok to leave the` super`?

Reply:
Sorry, I didn't mean to say to drop *just* that line, but the whole `def __init__()`. In this case, the function will not be overwritten, and so the "super" will be called instead, automatically.